portfolio/src/main/java/com/google/sps/servlets/data/PersonalDetails.java

```diff
@@ -1,19 +1,18 @@
 package com.google.sps.data;
-
-import java.util.Date;
+import java.util.Random;
 
 public final class PersonalDetails {
```

Comment:
Add a javadoc for the class (https://www.tutorialspoint.com/java/java_documentation.htm)